modules/javafx.graphics/src/main/java/javafx/animation/Interpolator.java

```diff
@@ -256,6 +266,69 @@ public static Interpolator TANGENT(Duration t, double v) {
         return new NumberTangentInterpolator(t, v);
     }
 
+    /**
+     * Specifies the step position of a step interpolator.
+     * <p>
+     * The step position determines the location of rise points in the input progress interval, which are the
+     * locations on the input progress axis where the output progress value jumps from one step to the next.
+     *
+     * @since 23
+     */
+    public enum StepPosition {
+        /**
+         * The interval starts with a rise point when the input progress value is 0.
+         */
+        START,
+
+        /**
+         * The interval ends with a rise point when the input progress value is 1.
+         */
+        END,
+
+        /**
+         * All rise points are within the open interval (0..1).
+         */
+        BOTH,
```

Comment:
Looks like the docs for BOTH and NONE are swapped? (this would also affect the CSR)

Reply:
Good catch! I've also updated the CSR.